maven-plugins/enforcer-maven-plugin/src/main/java/io/helidon/build/maven/enforcer/GitCommands.java

```diff
@@ -326,7 +333,15 @@ private static GitOperation gitOp(String line) {
         if (line.startsWith("R")) {
             return GitOperation.RENAME;
         }
-
+        if (line.startsWith("T\t")) {
```

Comment:
Can you make this logic testable ?

I.e. Separate the process execution and the parsing of its output from the gitTracked method so that we can test the parsing by feeding text to it.

Maybe something like this:

```java
static Map<String, Integer> gitLog(InputStream is) {
    // parsing logic
}
```